app/Integrations/Monzo/MonzoPlugin.php

```
@@ -0,0 +1,775 @@
+            ]
+        );
+
+        // Add balance blocks (Spend Today + Balance Change)
+        try {
+            $this->addBalanceBlocks($event, $integration, $account, $date, $balance, $spendToday);
+        } catch (\Throwable $e) {
+            // Non-fatal if blocks fail
+        }
+    }
+
+    public function addBalanceBlocks(Event $event, Integration $integration, array $account, string $date, int $balance, int $spendToday): void
```

PHPMD (2.15.0)

Avoid unused parameters such as '$account'. (Unused Code Rules)

(UnusedFormalParameter)